lib/commons/matches/index.js

```diff
@@ -0,0 +1,51 @@
+/* exported matches */
```

Comment:
At some point maybe. I've been thinking about potentially using them as rule selectors. But while they are commons? We didn't document our other commons, except through JSDocs (which I'll do), so I don't think that needs to be in this PR. Happy to raise an issue for it though so we know to do it later.

Reply:
We have introduced a npm script `npm run api-docs`, which generates jsdoc's based API documentation. This can be accessed via `http://localhost/doc/api`.

See - https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#full-api-reference-for-developers

Reply:
Like Jey said, collaborators can generate docs for commons.